erpnext/setup/doctype/naming_series/naming_series.py

```diff
@@ -131,9 +136,13 @@ def validate_series_name(self, n):
 			throw(_('Special Characters except "-", "#", "." and "/" not allowed in naming series'))
 
 	def get_options(self, arg=None):
+		global x
```

Comment:
use the local variables instead of global, please use the readable variable name